lib/handler/status/durations.c

```diff
@@ -64,6 +69,11 @@ static void durations_status_per_thread(void *priv, h2o_context_t *ctx)
         ADD_DURATION(response_time);
         ADD_DURATION(total_time);
 #undef ADD_DURATION
+        if (agg_stats->stats.evloop_latency_nanosec.capacity <= ctx->thread_index) {
+            h2o_vector_reserve(NULL, &agg_stats->stats.evloop_latency_nanosec, ctx->thread_index + 1);
+        }
+        agg_stats->stats.evloop_latency_nanosec.entries[ctx->thread_index] = h2o_evloop_get_execution_time(ctx->loop);
```

Comment:
Doesn't `h2o_evloop_get_execution_time` return time by milliseconds? I think you might want to create `h2o_evloop_get_execution_time_nanosec` and call that function.

Reply:
That's correct. Sorry that this line is in a weird state. The plan precisely is to assign a nanosec value from a new func that will return the execution time in nanosec.